### PR TITLE
Added content to user-docs.

### PR DESCRIPTION
**Pull request by breakreturn**
An initial description of the launcher responsibilities
when integrating with SC.

### CODE REVIEW

**Review by DunderRoffe — APPROVED** [review]

File: doc/chapters/integration-guidelines/index.rst [hunk 1]
```diff
@@ -84,3 +80,30 @@ Here is an example manifest defining Wayland access::
...
+:ref:`Design <design>`.
+
+The assumed scenario in this section is that a launcher want to start an application inside the container.
+
+The launcher should do the following:
+
+ * Make the app home directory available inside the container.
+ * Set the HOME environment variable in the container point to the above directory.
+
+The above actions are performed by interacting with the SoftwareContainerAgent :ref:`D-Bus API <api>`.
```
Comment (tobsan) [inline doc/chapters/integration-guidelines/index.rst:98]: This ties in closely to the docs describing the launcher in the design chapter. Are there opportunities for consolidation?

**Review by tobsan — APPROVED** [review]
Looks good in general.

**Comment by e8johan** [comment]
The commit message is a bit non-descript, but the contents is great.

File: doc/chapters/integration-guidelines/index.rst [hunk 2]
```diff
@@ -84,3 +80,30 @@ Here is an example manifest defining Wayland access::
...
 
+
+The role of a launcher
+======================
+
+This section describes what typical integration actions are needed to integrate SoftwareContainer with
+a launcher. For an overview of the general architecture involving a launcher and SoftwareContainer, see
+:ref:`Design <design>`.
+
+The assumed scenario in this section is that a launcher want to start an application inside the container.
```
Comment (thenor) [inline doc/chapters/integration-guidelines/index.rst:91]: s/want/wants

Comment (thenor) [reply doc/chapters/integration-guidelines/index.rst:91]: Also "the launcher" can be used instead of "a".

**Pull request merged**